packages/main/src/ComboBoxPopover.hbs

```diff
@@ -2,6 +2,7 @@
 	no-arrow
 	content-only-on-desktop
 	_disable-initial-focus
+	id="{{_id}}-responsive-popover"
```

Comment:
This is in a separate shadow DOM tree, won't work. You can't reference it from within the shadow tree of the main component

Reply:
You are right. We will remove the aria-controls